llvmlite/binding/initfini.py

```diff
@@ -1,7 +1,7 @@
-from ctypes import c_uint
+from ctypes import byref, c_bool, c_uint, POINTER, c_char_p, c_int
```

Comment:
What do the changes in this file do?

Reply:
Probably nothing. The original `lld_main()` function was put here by the original author of it. I moved it but forgot to remove the unused imports from `initfini.py`